lib/glific/messages.ex

```diff
@@ -246,7 +246,11 @@ defmodule Glific.Messages do
              interactive_template_id
            ),
          interactive_content <-
-           InteractiveTemplates.get_translations(interactive_template, language_id),
+           InteractiveTemplates.get_translations(interactive_template, language_id)
+           |> InteractiveTemplates.get_clean_interactive_content(
```

Comment:
Can't we move this function to get_translations itself?

Reply:
@pankaj-ag I thought of the same thing but then the function get_translation definition changes so I intentionally made a different function. Let me know your thoughts on same